File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/DiligenciaAtividadeVM.cs
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class DiligenciaAtividadeVM
	{
		public string CodigoSubclasse { get; set; }
		public string CodigoSubclasseAtividade { get; set; }
		public string DescricaoSubclasse { get; set; }
		public int? IdDiligencia { get; set; }
		public int? IdDiligenciaAtividade { get; set; }
		public int? IdPessoaJuridica { get; set; }
		public int? IdProtocolo { get; set; }
		public bool IsAtividadeExercida { get; set; }
		public IEnumerable<DiligenciaAtividadeSetorVM> Setor { get; set; }
	}
}

File: src/Suframa.Sciex.UserInterface/www/app/view-model/DiligenciaAtividadeVM.ts
import { diligenciaAtividadeSetorVM } from './diligenciaAtividadeSetorVM';

export class diligenciaAtividadeVM {
	codigoSubclasse: string;
	codigoSubclasseAtividade: string;
	descricaoSubclasse: string;
	idDiligencia?: number;
	idDiligenciaAtividade?: number;
	idPessoaJuridica?: number;
	isAtividadeExercida: boolean;
	setor: diligenciaAtividadeSetorVM;
}
